apps/src/codebridge/FileTabs/FileTabs.tsx

```diff
@@ -77,7 +88,12 @@ export const FileTabs = React.memo(() => {
       >
         <SortableContext items={files} strategy={horizontalListSortingStrategy}>
           {files.map(f => (
-            <Sortable id={f.id} key={f.id} isDragging={f.id === draggingFileId}>
+            <Sortable
+              id={f.id}
+              key={f.id}
+              isDragging={f.id === draggingFileId}
+              onKeyDown={event => handleTabActivation(event, f.id)}
+            >
               <FileTab file={f} />
```

Comment:
Confirming that we're good with a user selecting 'm' key to enter move mode while on the 'close' x button or the tab label.

Reply:
I don't know that there's a way around it with the current layout (we could consider refactoring). @hannahbergam what do you think?

Reply:
I think that's okay! I can see an interaction where a user is considering closing a tab but then decides to move it somewhere else instead so it doesn't seem like a sequence of events we specifically want to disallow